src/modelfree/score_agent.py

```diff
@@ -24,7 +24,8 @@ def announce_winner(sim_stream):
                 yield game_outcome(info)
 
 
-def get_empirical_score(_run, env, agents, episodes, render=False):
+def get_empirical_score(_run, env, agents, episodes, render=False,
+                        record_traj=False, traj_dir='data/'):
```

Comment:
Yuck! Record trajectory definitely has no job here. This will also be unnecessary if we make it a wrapper.